tests/core/test_validator.py

```diff
@@ -16,7 +16,7 @@
         domain_path=DEFAULT_DOMAIN_PATH,
         training_data_paths=[DEFAULT_NLU_DATA, DEFAULT_STORIES_FILE],
     )
-    return await Validator.from_importer(importer)
+    return await Validator.from_importer(importer, out="log.out")
```

Comment:
there should be a test which checks the file is created and has the right content